src/OpenTelemetry.Extensions.Hosting/OpenTelemetryServicesExtensions.cs

```diff
@@ -27,6 +30,29 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// </summary>
 public static class OpenTelemetryServicesExtensions
 {
+    /// <summary>
+    /// Adds OpenTelemetry SDK services into the supplied <see
+    /// cref="IServiceCollection"/>.
+    /// </summary>
+    /// <remarks>
+    /// Note: This is safe to be called multiple times and by library authors.
+    /// Only a single <see cref="TracerProvider"/> and/or <see
+    /// cref="MeterProvider"/> will be created for a given <see
+    /// cref="IServiceCollection"/>.
+    /// </remarks>
+    /// <param name="services"><see cref="IServiceCollection"/>.</param>
+    /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining
+    /// calls.</returns>
+    public static OpenTelemetryBuilder AddOpenTelemetry(this IServiceCollection services)
+    {
+        Guard.ThrowIfNull(services);
+
+        services.TryAddEnumerable(
+            ServiceDescriptor.Singleton<IHostedService, TelemetryHostedService>());
```

Comment:
All this effort for this one dumb line 🤣

Reply:
Do we need a log here if the TryAdd fails?

Reply:
`TryAddEnumerable` (oddly) does not return true or false so we have no way of knowing. But even then, it only no-ops if an exact march for service type (IHostedService) and implementation type (TelemetryHostedService) is found. So it would only no-op for repeated calls to `AddOpenTelemetry` which is totally fine.